src/main/java/org/apache/geronimo/microprofile/impl/jwtauth/servlet/GeronimoJwtAuthFilter.java

```diff
@@ -42,17 +43,27 @@
     private String headerName;
     private String cookieName;
     private String prefix;
+    @Inject
     private JwtParser service;
+    @Inject
     private GeronimoJwtAuthExtension extension;
     private Collection<String> publicUrls;
+    @Inject
+    private GeronimoJwtAuthConfig config;
 
     @Override
     public void init(final FilterConfig filterConfig) {
         final CDI<Object> current = CDI.current();
```

Comment:
Should be init to null and lazily initialized no?

Reply:
I cleanup the handling of CDI.current in Aries CDI and managed to solve this so I don't need it. removed!